src/renderers/common/Textures.js

```diff
@@ -326,6 +326,14 @@ class Textures extends DataMap {
 
 			this.info.memory.textures ++;
 
+			//
+
+			if ( texture.isVideoTexture && texture.colorSpace !== SRGBColorSpace ) {
```

Comment:
I think you may want to check for the sRGB transfer functions, not color space. For example Display P3 uses sRGB transfer functions, and should work just fine assuming an appropriate working color space. Example:

```suggestion
			if ( texture.isVideoTexture && ColorManagement.getTransfer( texture.colorSpace ) !== SRGBTransfer ) {
```

Reply:
> Would we not allow putting data into a video, annotated with NoColorSpace?

I initially considered this use case, so the warning only mentioned **rare** scenarios...